README.md

```diff
@@ -1 +0,0 @@
-# DeployPythonApp
```

Comment:
you already told that your main branch should only have one commit ie. Initial commit, why you still pushed second commit on it and made changes to README.md ?